docs/rules/prop-types.md

```diff
@@ -29,6 +29,29 @@ function Hello({ name }) {
 }
 ```
 
+Examples of correct usage without warnings:
+
+```jsx
+var Hello = React.createClass({
+  propTypes: {
+    name: React.PropTypes.string.isRequired,
+  },
+  render: function() {
+    return <div>Hello {this.props.name}</div>;
+  }
+});
+
+// Or in ES6:
+class Hello extends React.Component {
+  static propTypes = {
```

Comment:
this isn't actually ES6 - it's using a stage 2 proposal for public class properties.

The current standard JS way of doing this is `Hello.propTypes = { … }`, after the class definition.